modules/sdk-core/src/bitgo/keychain/iKeychains.ts

```diff
@@ -30,6 +30,8 @@ export interface Keychain {
   prv?: string;
   provider?: string;
   encryptedPrv?: string;
+  // Required for MPCV2 keys where we reduce the amount of data needed for the keycard
+  reducedEncryptedPrv?: string;
```

Comment:
Nit: Add a note that this is not stored in the db, and generated client side when creating the keys.

Reply:
done